README.md

```diff
@@ -21,17 +21,69 @@ dependencies, you need to require
 }
 ```
 
-Please, read the website to [get more informations about how to
-install](http://hoa-project.net/Source.html).
+And add `RulerBundle` to your `AppKernel`:
+
+```php
+// app/AppKernel.php
+
+class AppKernel extends Kernel
+{
+    // …
+
+    public function registerBundles()
+    {
+        $bundles = array(
+            // …
+            new Hoathis\Bundle\RulerBundle\HoathisRulerBundle(),
+        );
+
+        // …
+    }
+}
+```
 
 ## Quick usage
 
-…
+### Ruler service
 
-## Documentation
+The `RulerBundle` defines a `hoathis.ruler` service that can be used as
+described in [Ruler's documentation](http://hoa-project.net/En/Literature/Hack/Ruler.html).
+
+```php
+public function indexAction(Request $request)
+{
+    // 1. Write a rule.
+    $ruler = $this->get('hoathis.ruler');
+    $rule  = 'group in ["customer", "guest"] and points > 30';
 
-Different documentations can be found on the website:
-[http://hoa-project.net/](http://hoa-project.net/).
+    // 2. Create a context.
+    $context           = new \Hoa\Ruler\Context();
```

Comment:
You're right, I also think that a factory method like `getNewContext` directly exposed by Hoa\Ruler would be better.
Having a service for the context does not make sense but having the same API for both the bundle and the standalone library does :+1: 

The example would become:

```php
public function indexAction(Request $request)
{
    // 1. Write a rule.
    $ruler = $this->get('hoathis.ruler');
    $rule  = 'group in ["customer", "guest"] and points > 30';

    // 2. Create a context.
    $context           = $ruler->getNewContext();
    $context['group']  = 'customer';
    $context['points'] = function () {
        return 42;
    };

    // 3. Assert!
    var_dump(
        $ruler->assert($rule, $context) // bool(true)
    );
}
```

I'll update the README when Hoa\Ruler has this kind of method :)